pkgs/by-name/ho/homebridge-config-ui-x/package.nix

```
@@ -0,0 +1,65 @@
+{
+  lib,
+  pkgs,
+  buildNpmPackage,
+  fetchFromGitHub,
+  fetchNpmDeps,
+  npmHooks,
+  nodejs_22,
+}:
+
+let
+  version = "4.71.2";
+  src = fetchFromGitHub {
+    owner = "homebridge";
+    repo = "homebridge-config-ui-x";
+    rev = "v${version}";
```

Use `tag` instead (this also goes for the homebridge package):

```suggestion
    tag = "v${version}";
```